comeet/UI/MainMenu/MainMenuViewModel.swift

```diff
@@ -26,14 +26,15 @@ class MainMenuViewModel : BaseViewModel {
     
     func fetchMeetings() {
         let start = ""
-        print(start)
+        print("LOADING MEETINGS....")
         let end = ""
         
         
         
         fetcher.getMeetings(organization: authenticator.getOrganization(), user: "", start: start, end: end) { [weak self] (meetings: [Meeting]?, error: Error?) in
             if let meetings = meetings {
                 
+                print("COMPLETE... PRESS RELOAD")
```

Comment:
You should use the reloadBinding property closure. And on the viewController set the property. So that every time it is called from the viewModel the UI reloads accordingly.